### DISCUSSION TITLE
Enable Api endpoint for integrate with pos or develop mobile apps

### DISCUSSION THREAD

**Discussion by nzian** [innocommerce/innoshop#62] [Ideas]
Initial impression is good project. I like it so far also the structure and most importantly the feature list. Do you have plan to enable api in this project. If you have plan or share anyhing what you thinking about?
Thank you  again for the nice project.

**Comment by yushine** [comment]
@nzian  In fact, the project has integrated backend interfaces and a complete set of frontend APIs, which can provide a comprehensive shopping experience for APPs. We have already developed a complete mobile APP using these APIs, and you can use them to develop your own APP or headless Web applications.

The code for the frontend APIs can be found in the following GitHub repository:
[InnoShop FrontApiControllers](https://github.com/innocommerce/innoshop/tree/master/innopacks/restapi/src/FrontApiControllers)

For detailed information about the system architecture, please refer to the documentation:
[System Architecture | InnoShop](https://docs.innoshop.cn/en/introduction/architecture.html)

Additionally, you can view the detailed API documentation:
[Innoshop Frontend API](http://front-api.innoshop.cn/)

**Reply by nzian** [reply]
I will say this is really really nice work. Api doc is really super attractive. organised and described sample code for multiple languages. If possible I will give double star.
Thank you for quick reply.
👍
